### PR TITLE
fix(markdown): fix error of option setting

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [ ] Updated Storybook documents for component
- [ ] Wrote test codes for composable, utils and etc.
- [ ] Tested with console(if usages are changed) 

### Description
SSIA

### CODE REVIEW

**Review by piggggggggy — APPROVED** [review]
L G T M

**Review by song2kim — APPROVED** [review]
LGTM~

File: src/data-display/dynamic/dynamic-layout/templates/markdown/index.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@ import {
     computed, getCurrentInstance,
     reactive, toRefs,
 } from 'vue';
-import type { Vue } from 'vue/types/vue';
+import type Vue from 'vue';
```
Comment (minseolee) [inline src/data-display/dynamic/dynamic-layout/templates/markdown/index.vue:18]: Most of codes are working by the previous code. 
Is it ok ?

**Review by minseolee — APPROVED** [review]
LGTM~!

File: src/data-display/dynamic/dynamic-layout/templates/markdown/index.vue (hunk 1 above)
Comment (WANZARGEN) [reply src/data-display/dynamic/dynamic-layout/templates/markdown/index.vue:18]: Yes. My IDE returns error if import from 'vue/types/vue'

**Pull request merged**